.travis.yml

```
@@ -3,6 +3,8 @@ version: ~> 1.0
 import:
   - silverstripe/silverstripe-travis-shared:config/provision/self-jobs-range.yml
 
+# Require installer which will include the simple theme for a few functional tests, otherwise the
+# "Welcome to Silverstripe" no theme fallback template is shown which will break the tests
 env:
   global:
     - REQUIRE_INSTALLER="*"
```

Do you still want to do this to make it clearer what we are installing?

```suggestion
    - REQUIRE_INSTALLER="^4"
```

---

I'm not fussed at this point